### ISSUE TITLE
[Bug]: [Auto layout] Container like widgets are not resizable

### ISSUE THREAD

**Issue by Sripriya93** [appsmithorg/appsmith#21259]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description

When we drag and drop a container like widget and try to resize the height of the widget in Auto layout using the signifier we are able to see the widget automatically collapses or expands to he original size.

https://user-images.githubusercontent.com/104895602/223679634-58a97bd5-e865-4c89-bad8-2cc2f33940da.mp4

### Steps To Reproduce

1. DnD a container like widget
2. Try to expand or reduce the height of the widget
3. Observe the widget collapses or expands to its original size.

### Environment

Deploy Preview

### Version

Appsmith Community v1.9.10-SNAPSHOT

**Comment by Sripriya93** [comment]
Waiting on decision if we need vertical resizers or Auto height. QA on hold.

**Comment by marks0351** [comment]
@Sripriya93 can you pls qa this now.

**Comment by Sripriya93** [comment]
Vertical resizer is not used anymore. We can close this issue @marks0351
